lib/routes/xbookcn/blog.ts

```diff
@@ -0,0 +1,65 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '/:label?/:max?', // 路由路径
+    categories: ['other'], // 分类
+    example: '/xbookcn/:label?', // 示例路径
+    parameters: { label: '按名称分类，详见https://blog.xbookcn.net/p/all.html', max: '最大条数' }, // 参数
```

Comment:
Please use the common parameter `limit` instead.
https://docs.rsshub.app/guide/parameters#limit-entries